packages/server/customer-os-api/test/neo4j/neo4j_data_helper.go

```diff
@@ -845,6 +849,16 @@ func InteractionEventPartOfInteractionSession(ctx context.Context, driver *neo4j
 	})
 }
 
+func InteractionEventRepliesToInteractionEvent(ctx context.Context, driver *neo4j.DriverWithContext, interactionEventId, repliesToInteractionEventId string) {
+	query := "MATCH (ie:InteractionEvent {id:$interactionEventId}), " +
```

Comment:
please use the tenant in selection

Reply:
This is code only used in unit tests and you notice none of the methods currently currently put the tennant lablel,

if you want the full test lib refactored, you probably should talk to Alex

Reply:
I've added tenant to this query

Though this lib is only for provisioning in unit tests, there are plenty of other methods out there that don't use tennant

if you want a full refactoring of this test lib you should talk to alex